tracer/test/Datadog.Trace.Tools.Runner.IntegrationTests/ConfigureCiCommandTests.cs

```diff
@@ -0,0 +1,75 @@
+// <copyright file="ConfigureCiCommandTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using FluentAssertions;
+using Xunit;
+
+namespace Datadog.Trace.Tools.Runner.IntegrationTests
+{
+    [Collection(nameof(ConsoleTestsCollection))]
+    public class ConfigureCiCommandTests
+    {
+        [Fact]
+        public void ConfigureCi()
+        {
+            var commandLine = "ci configure azp --dd-env TestEnv --dd-service TestService --dd-version TestVersion --tracer-home TestTracerHome --agent-url TestAgentUrl";
+
+            using var console = ConsoleHelper.Redirect();
+
+            var result = Program.Main(commandLine.Split(' '));
+
+            result.Should().Be(0);
+
+            var environmentVariables = new Dictionary<string, string>();
+
+            foreach (var line in console.ReadLines())
+            {
+                // ##vso[task.setvariable variable=DD_DOTNET_TRACER_HOME]TestTracerHome
+                var match = Regex.Match(line, @"##vso\[task.setvariable variable=(?<name>[A-Z1-9_]+)\](?<value>.*)");
+
+                if (match.Success)
+                {
+                    environmentVariables.Add(match.Groups["name"].Value, match.Groups["value"].Value);
+                }
+            }
+
+            environmentVariables["DD_ENV"].Should().Be("TestEnv");
+            environmentVariables["DD_SERVICE"].Should().Be("TestService");
+            environmentVariables["DD_VERSION"].Should().Be("TestVersion");
+            environmentVariables["DD_DOTNET_TRACER_HOME"].Should().Be("TestTracerHome");
+            environmentVariables["DD_TRACE_AGENT_URL"].Should().Be("TestAgentUrl");
```

Comment:
As above:
```suggestion
            environmentVariables.Should().Contain("DD_ENV", "TestEnv");
            environmentVariables.Should().Contain("DD_SERVICE", "TestService");
            environmentVariables.Should().Contain("DD_VERSION", "TestVersion");
            environmentVariables.Should().Contain("DD_DOTNET_TRACER_HOME", "TestTracerHome");
            environmentVariables.Should().Contain("DD_TRACE_AGENT_URL", "TestAgentUrl");
```

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/2295/commits/54c8e313695aae194ab141e73e2b033a62f3f06b